src/xbot/common/controls/sensors/MockGyro.java

```diff
@@ -1,11 +1,13 @@
 package xbot.common.controls.sensors;
 
 import xbot.common.controls.MockRobotIO;
+import xbot.common.controls.sensors.navx.AHRS;
```

Comment:
Andy, the AHRS (attitude and heading reference system) should only be used in the InertialMeasurementUnitAdapter class.

In this mock class, you have to "mock" (a.ka. "fake") the behavior of the AHRS.